source/_docs/composer-drupal-8.md

```diff
@@ -79,7 +79,9 @@ If Pantheon is your site's only repository, then you do have to commit the `vend
 
 ### Avoid Committing Dependencies as Submodules
 
-Sometimes running Composer commands like `install`, `update`, or `require` result in the downloading of `.git` directories for the dependencies being downloaded. Downloading the Git source of a dependency may be helpful for editing and contributing back to a dependency, but it is unlikely to be needed for most site building. Use `--prefer-dist` when running `composer require` so that Composer will download a distribution of the dependency that does not include the Git history of that dependency. When you `git add` the dependency, Git will not attempt to add the dependency as [a git submodule, which is not supported on Pantheon](https://pantheon.io/docs/git-faq/#does-pantheon-support-git-submodules%3F).
+Sometimes running Composer commands like `install`, `update`, or `require` result in the downloading of `.git` directories for the dependencies being downloaded. This happens when a development (non-dist) version of a module is selected. Downloading the Git source of a dependency may be helpful for editing and contributing back to a dependency, but it is unlikely to be needed for most site building, and actually causes problems when committing these directories.
```

Comment:
How about "development version (a release without a specific version tag)"?

There might also be a question in some people's mind on "what is a version tag", maybe.